appcelerator/titanium_mobile
# [TIMOB-15116]iOS: Adding support for 64 bit iOS7 4-inch simulator.

**TESTING INSTRUCTIONS**
Code Review of the ios-sim project to be done on https://github.com/appcelerator/ios-sim/pull/5

the testing can be done only through CLI. 
- Go to any Titanium app folder in terminal or create a new app.
- "ti build -p ios -Y iphone -S 7.0 --retina --tall --sim-64bit"
- Kill the ios simulator app. 
- Test out variation of the --retina, --tall and simulator versions and sim types for regression testing.

**Make sure to kill the simulator before running the next command. Otherwise the right simulator will not be displayed**
